src/utils.ts

```diff
@@ -93,3 +93,8 @@ export function isWithinRootPath(file: string) {
 
 	return workspace.rootPath != null && file.startsWith(workspace.rootPath + path.sep); 
 }
+
+export function log(message: any): void {
+	if (config.verbose)
```

Comment:
I wonder if there's any value in `config.versbose` anymore. The console (presumably) only logs with the debugger attached so I don't think there's much value in turning it off? What's left is very low traffic (and we do some logging without checking this, like warnings in error responses, so maybe it's also applied a bit inconsistently).